gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-common/src/main/java/io/gravitee/am/gateway/handler/common/email/impl/EmailServiceImpl.java

```diff
@@ -253,6 +252,7 @@ private Map<String, Object> prepareEmailParams(User user, Client client, Integer
         params.put("token", token);
         params.put("expireAfterSeconds", expiresAfter);
         params.put("domain", new DomainProperties(domain));
+        params.put("gatewayHost", domainService.gatewayUrl().getHost().replaceFirst("^www\\.", ""));
```

Comment:
**question**: As it is not possible to use variable in the template for the "from" header, should we keep this variable :thinking: ? see https://github.com/gravitee-io/gravitee-access-management/pull/3275/files#diff-18fe570831b203224bdfd7400439c0778b685600a6ac699c057031f89c5c20afR13